### ISSUE TITLE
Enable Celery when running via gravity

### ISSUE THREAD

**Issue by mvdbeek** [galaxyproject/gravity#38]
I don't think we can switch this to true within Galaxy just now, but we know in gravity that Celery is available, so we could set `GALAXY_CONFIG_ENABLE_CELERY_TASKS=true` in the supervisor config.

**Comment by nsoranzo** [comment]
I think activating this via the `GALAXY_CONFIG_` environment variable may confuse the admins, since it overrides what's in `galaxy.yml` .

**Comment by nsoranzo** [comment]
Right, but it would still enable it if it's left commented in `galaxy.yml`

**Comment by mvdbeek** [comment]
Yes, that is the idea. I prefer this over rewriting the Galaxy config, and otherwise we'll have to come up with some other solution to checking if celery is available and functional.

**Comment by natefoo** [comment]
I get the point, an admin may be confused why the commented-out-default is not actually the applied setting in this case. Gravity could print a warning that it's doing this I guess? But it would also hopefully be fairly obvious from the fact that it starts celery processes.
